src/providers/ogr/qgsogrprovider.cpp

```diff
@@ -2425,6 +2425,9 @@ QString createFilters( const QString &type )
     bool kmlFound = false;
     bool dwgFound = false;
     bool dgnFound = false;
+
+    sProtocolDrivers += QLatin1String( "HTTP/HTTPS/FTP,vsicurl;AWS S3,vsis3;Google Cloud Storage,vsigs;Microsoft Azure Blob,vsiaz;Alibaba Cloud OSS,vsioss;OpenStack Swift Object Storage,vsiswift;" );
```

Comment:
I don't think this is needed. Those are not drivers

Reply:
@rouault , what do you mean by "move that list closer to the UI"?

Reply:
Your move to qgsogrsourceselect.cpp satisfies this.